src/components/movielist.css

```diff
@@ -0,0 +1,92 @@
+.movielist {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(25%, 1fr));
+    grid-gap: 0px;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    overflow-x: hidden;
+}
+
+.movielist li {
+    margin: 0;
+    padding: 0;
+}
+
+.movie-image:hover::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: rgba(0, 0, 0, 0.5);
+    /* Dark overlay */
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    color: white;
+    font-size: 1.2rem;
+}
+
+
+.movie-image {
+    position: relative;
+    overflow: hidden;
+}
+
+
+.movie-image img {
+    width: 100%;
+    height: auto;
+    /* Make sure the image covers the entire container */
+}
+
+
+.movie-title {
+    position: absolute;
+    font-family: 'Montserrat', sans-serif;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    color: white;
+    font-size: 30px;
+    text-align: left;
+    padding: 15px;
+    transition: opacity 0.3s;
+    opacity: 0;
+    margin-bottom: 40px;
+}
+
+.release-date {
+    position: absolute;
+    font-family: 'Montserrat', sans-serif;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    color: white;
+    font-size: 15px;
+    text-align: left;
+    padding: 15px;
+    transition: opacity 0.3s;
+    opacity: 0;
+    margin-bottom: 10px;
+}
+
+
+.movie-image:hover .movie-title,
+.movie-image:hover .release-date {
+    opacity: 1;
+    /* Show the title on hover */
+}
+
+
+/* Mobile */
+@media (max-width: 471px) {
```

Comment:
no media query for bigger screens or is it implied that it uses the fulls screen?